src/azure-cli/HISTORY.rst

```diff
@@ -15,6 +15,10 @@ Release History
 
 * az network private-dns link vnet create/update: Fixes #9851. Support cross-tenant virtual network linking.
 
+**CosmosDB**
+* Add parameter for sql container create and update command
```

Comment:
Can you describe your command change in detail? Besides sql containter, you also add a parameter for gremlin and declare the parameter name here.

In addition, you need to rearrange your release note to make all cli modules in alphabetical order here.